src/pages/export-page/ExportPage.tsx

```diff
@@ -17,6 +17,14 @@ export const ExportPage = () => {
     loadedTimetableEvents,
   } = useFormContext();
 
+  const updatedTimetableEvents = useMemo(() => {
+    return timetableEvents.map((x) =>
+      x.developmentPlanEvent == "timetable-updated"
+        ? { ...x, eventDate: new Date().toISOString() }
```

Comment:
do we want to use getFormattedLongDate() here? (in that case keep the export)

Reply:
not necessarily for this PR but we should have the DateInput deal with dates that might come in the format that toIsoString() returns (i.e. 2024-02-29T00:00:00.000Z). I think at the moment we are doing a split("-") in DateInput which will make day = 29T00:00:00.000Z instead of just 29